fibbingnode/southbound/lsdb.py

```diff
@@ -466,7 +503,29 @@ def process_lsa(self):
                                  for part in lsa_info.split(SEP_GROUP) if part]
                     hdr = LSAHeader.parse(lsa_parts.pop(0))
                     lsa = LSA.parse(hdr, lsa_parts)
+                    new_seqnum = hdr.lsa_seqnum
+
                     log.debug('Parsed %s: %s', action, lsa)
+
+                    # For now, we just log lsa seq_num and ignore an LSA when
+                    # it has an older seq_num than the one in the database
+                    c_seqnum = self.get_current_seq_number(lsa)
+                    if not c_seqnum:
+                        log.debug('LSA was not in the LSDB yet')
+                    else:
+                        log.debug('Seqnum for LSA in lsdb: %s)', c_seqnum)
+                        log.debug('Seqnum for newly arrived LSA: %s)', new_seqnum)
+                        if new_seqnum == c_seqnum:
+                            log.debug("DUPLICATE seqnum for new LSA: %s", lsa)
+
+                        elif self.is_newer_seqnum(new_seqnum, c_seqnum):
+                            log.debug("NEWER seqnum received")
+
+                        else:
```

Comment:
This needs to become:
elif hdr.age < MAX_LS_AGE:

in order to support premature LSA aging to wrap the sequence number around